### PR TITLE
enh(c-like) Fix incorrect highlighting for interger suffix #2919

### LINKED ISSUE #2919: C/C++ Incorrect highlighting for LL suffix
**Describe the issue**
Only the first `L` is highlighted when the `LL` integer literal suffix is used

**Which language seems to have the issue?**
C and C++

**Are you using `highlight` or `highlightAuto`?**
I don't know. I've just seen the issue on StackOverflow
...

**Sample Code to Reproduce**

No code, but some questions where the issue occurs

- https://stackoverflow.com/questions/15575054/what-does-ll-mean
- https://stackoverflow.com/questions/33377358/integer-overflow-in-boolean-expressions/33377537#33377537
- https://stackoverflow.com/questions/65318714/expression-evaluation-in-32-bit-cpu

Screenshot:

[Screen Shot 2020-12-16 at 3 36 59 PM](https://stackoverflow.com/questions/33377358/integer-overflow-in-boolean-expressions/33377537)
 
**Expected behavior**
Both Ls should be highlighted

**Additional context**

### PR DESCRIPTION
**Pull request by il3ven**
This pull request enhances the detection of integer suffixes such as LL, ll, ul in clike languages.

Resolves #2919 

### Changes
The regular expression in c-like.js has been updated in order to detect the following suffixes.
image
**Source:** https://www.learncpp.com/cpp-tutorial/literals/

### Checklist
- [x] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`
- [ ] Added myself to `AUTHORS.txt`, under Contributors

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Grammar improvements:
 - enh(dart) Fix empty block-comments from breaking highlighting (#2898) [Jan Pilzer][]
 - enh(dart) Fix empty doc-comment eating next line [Jan Pilzer][]
 - enh(asciidoc) Adds support for unconstrained bold syntax (#2869) [Guillaume Grossetie][]
+- enh(c-like) Incorrect highlighting for interger suffix (#2919) [Vaibhav Chanana][]
```
Comment (joshgoebel) [inline CHANGES.md:42]: You also need to add yourself around line ~70 so it knows where to link to.

**Review by joshgoebel — APPROVED** [review]
Please add link to CHANGES.md.  Otherwise this looks good. Thanks!

File: CHANGES.md (hunk 1 above)
Comment (il3ven) [reply CHANGES.md:42]: My name is already there.

Comment (joshgoebel) [reply CHANGES.md:42]: Oh indeed. :) I forget who is first-time contributor sometimes.

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
@il3ven Thanks!

**Pull request merged**